app/front/scripts/services/settings/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
generally speaking we can probably remove this module and get embed the settings object in the page (like it's done in the packager), no?